assets/_scss/components/_forms.scss

```diff
@@ -23,7 +23,7 @@ form {
     margin-bottom: 1.5em;
   }
 
-  input[name="password"] {
+  input[name="password"], input[name="confirmPassword"] {
```

Comment:
@msecret it may be good to keep it in a form to ensure people markup forms correctly. We can always update the specificity issue later down the line, so going to go ahead with this for now.